conjure-java-core/src/test/java/com/palantir/conjure/java/JerseyServiceGeneratorTests.java

```diff
@@ -63,7 +63,8 @@ public void testServiceGeneration_exampleService_requireNotNullAuthHeadersAndReq
 
         for (Path file : files) {
             if (Boolean.valueOf(System.getProperty("recreate", "false"))) {
-                Path output = Paths.get("src/test/resources/test/api/" + file.getFileName() + ".jersey");
+                Path output = Paths.get("src/test/resources/test/api/"
+                        + file.getFileName() + ".jersey_require_not_null");
```

Comment:
well spotted inconsistency!

Reply:
I'll pull this out into a separate PR, it's not related to this feature, it just allowed `-Drecreate=true` not cause tests to pass

Reply:
PR is up: https://github.com/palantir/conjure-java/pull/169